test/driver.ts

```diff
@@ -19,8 +19,8 @@ import {TransactionReceipt} from "web3-core";
 export const BANNING_LOCK_TIMEOUT = 7*24*60*60;
 export const DEPLOYMENT_SUBSET_MAIN = "main";
 export const DEPLOYMENT_SUBSET_CANARY = "canary";
-export const CONFORMANCE_TYPE_GENERAL = "General";
-export const CONFORMANCE_TYPE_COMPLIANCE = "Compliance";
+export const COMPLIANCE_TYPE_GENERAL = "General";
+export const COMPLIANCE_TYPE_COMPLIANCE = "Compliance";
```

Comment:
can we change this to "identified"? compliance is not a compliance type. if we do this we are essentially implementing a fancy boolean

@OdedWx

Reply:
No longer relevant as we plan to represent as boolean